front/src/modules/simulationResult/components/SpaceTimeChart/useGetProjectedTrainOperationalPoints.ts

```diff
@@ -40,13 +59,44 @@ const useGetProjectedTrainOperationalPoints = (
         };
         const { operational_points } = await postPathProperties(pathPropertiesParams).unwrap();
 
-        setOperationalPoints(
-          operational_points as NonNullable<PathProperties['operational_points']>
-        );
+        let operationalPointsWithAllWaypoints = operational_points as NonNullable<
+          PathProperties['operational_points']
+        >;
```

Comment:
Orthogonal to this PR since it was like this in the original code, but can't this cast be simplified to a `operational_points!`?

Reply:
Oh yep good point

Reply:
Done

Reply:
Hm I don't see this change in the diff I think?